scripts/user_functions.js

```diff
@@ -0,0 +1,193 @@
+var password_field = $("#passwordField")[0],
+  confirm_password_field = $("#confirmPasswordField")[0],
+  email_field = $("#emailField")[0],
+  email_regex = /(.+)@(.+){2,}\.(.+){2,}/,
+  likeCount = 0;
+
+$(document).ready(function() {
+  var usr = localStorage.getItem("signed");
+  if (usr == "true") {
+    $(".login-buttons").hide();
+    $(".logged-buttons").show();
+  } else {
+    $(".login-buttons").show();
+    $(".logged-buttons").hide();
+  }
+  $("#logout").click(function(event) {
+    event.preventDefault();
+    alert("You are now logged out");
+    localStorage.setItem("signed", "false");
+    $(".login-buttons").show();
+    $(".logged-buttons").hide();
+  });
+  $(".likeClass").hide();
+  $(".readLess").hide();
+  if (window.location.href == "https://appucyriac.github.io/gadget/") {
```

Comment:
Hi @appucyriac 
Move this set of codes to the 'loadContent' function and remove teh url check

Reply:
I have made the changes